datasets/noaa/pipelines/noaa/pipeline.yaml

```
@@ -0,0 +1,957 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+  - type: bigquery_table
+    table_id: "ghcnd_by_year"
+    description: "noaaspc"
+  - type: bigquery_table
+    table_id: "ghcnd_countries"
+    description: "noaaspc"
+  - type: bigquery_table
+    table_id: "ghcnd_inventory"
+    description: "noaaspc"
+  - type: bigquery_table
+    table_id: "ghcnd_states"
+    description: "noaaspc"
+  - type: bigquery_table
+    table_id: "ghcnd_stations"
+    description: "noaaspc"
+  - type: bigquery_table
+    table_id: "gsod_stations"
+    description: "noaaspc"
+  - type: bigquery_table
+    table_id: "hurricanes"
+    description: "noaaspc"
+  - type: bigquery_table
+    table_id: "lightning_strikes_by_year"
+    description: "noaaspc"
+
+dag:
+  airflow_version: 2
+  initialize:
+    dag_id: noaa
+    default_args:
+      owner: "Google"
+      depends_on_past: False
+      start_date: '2021-03-01'
+    max_active_runs: 1
+    schedule_interval: "0 1 0 0 6"
```

Let's add a comment for clarity.
```suggestion
    schedule_interval: "0 1 0 0 6"  # Every Saturday at 1am server time
```